Source/AGXUnrealEditor/Private/AGXUnrealEditor.cpp

```diff
@@ -693,6 +748,17 @@ void FAGXUnrealEditorModule::InitializeAssets()
 	AGX_MaterialLibrary::InitializeLidarAmbientMaterialAssetLibrary();
 }
 
+void FAGXUnrealEditorModule::OnGrabModeCommand() const
+{
+	FAGX_GrabMode::Activate();
+}
```

Comment:
I'm not convinced, since the keyboard shortcut is a bit unstable, I can see how a user might press the shortcut multiple times. `ESC` will disable it.